src/context.pxi

```diff
@@ -334,8 +333,12 @@ cdef class Context:
         _capi.blContextFillRectD(&self._self, &rect._self)
 
     def fill_path(self, Path path):
-        _capi.blContextFillPathD(&self._self, &path._self)
-
+        cdef:
+            _capi.BLPoint point
+        point.x = 0.
+        point.y = 0.
+        _capi.blContextFillPathD(&self._self, &point, &path._self)
```

Comment:
The signature of `fill_path` looks like it might need to change to better match the upstream API?

Reply:
I thought so but I did not wanted to change the user interface. And also I did not really get the difference with a move_to instruction.